### PR TITLE
Add browser build

### PR DESCRIPTION
**Pull request by XVincentX**
This would solve #13

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+; Install the "EditorConfig" plugin into your editor to use
+
+root = true
+
+[*]
+charset = utf-8
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
```
Comment (XVincentX) [inline .editorconfig]: So that, regardless of editor, everybody will write code with a minimal ruleset.

File: .gitignore [hunk 2]
```diff
@@ -25,3 +25,5 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+
+dist
```
Comment (XVincentX) [inline .gitignore]: Not required anymore as it's generated and bundled in npm

File: .npmignore [hunk 3]
```diff
@@ -5,3 +5,4 @@ examples
 .eslintrc
 .npmignore
 gulpfile.js
+webpack.babel.js
```
Comment (XVincentX) [inline .npmignore]: This is not required in downloaded package

**Comment by dvlsg** [comment]
Couple of changes:

- I would drop the `.editorconfig`. We already have `eslint` to handle our stylings, which is considerably more robust for javascript specific styling. If you want to continue to use it locally, feel free to add it to the `.gitignore`, but I'd like to treat `eslint` as the source of truth for styling.
- Keep `dist` out of the `.gitignore` (as in, actually commit it to the repository). I'd actually like to keep those files in the repository (at least for now). I may change that in the future, but since they're currently available in the repo, lets keep them there for now.

Other than that, looks good! I'll merge it in afterwards. Thanks for your work!

**Comment by XVincentX** [comment]
Those comments should be addessed in https://github.com/dvlsg/async-csp/pull/14/commits/e8b0f938066718d99f7fc1c5ffa1623ee3b65174#diff-1e70daafb475c0ce3fef7d2728279182

**Comment by dvlsg** [comment]
Awesome, thanks! I'll merge it in, and work on publishing something out which will give you access to that umd file through npmcdn. Hopefully this weekend at some point (I have a few other things I want in the next release that aren't quite done yet).

**Pull request merged**